### PR TITLE
Migrate Routing Spec list to standard JTable sorting.

### PR DESCRIPTION
**Pull request by MikeNeilson**
## Problem Description

Partial fix of #683. 

Describe the problem you are trying to solve.

## Solution

Migrated to standard JTable sorting.

## how you tested the change

Manually sorted the table by different means. Verified correct entries opened or deleted.

## Where the following done:

- [ ] Tests. Check all that apply:
   - [ ] Unit tests created or modified that run during ant test.
   - [ ] Integration tests created or modified that run during integration testing
         (Formerly called regression tests.)
   - [ ] Test procedure descriptions for manual testing
- [ ] Was relevant documentation updated?
- [ ] Were relevant config element (e.g. XML data) updated as appropriate

If you aren't sure leave unchecked and we will help guide you to want needs changing where.

### CODE REVIEW

**Comment by MikeNeilson** [comment]
This was smaller than expected, the Sources table is doing the same thing and I will add similar changes to this PR.

**Comment by MikeNeilson** [comment]
Okay, that's enough for this PR. The equipment table is next.

We're also going to need a follow up of PRs to just correct behavior. In reviewing various GUI patterns and just looking at the code. The TableModels should either be where all the database operations go, or they should be a wrapper to a generic form of of such model (preferred any future GUI toolkit changes). Currently the Panels, and even some upstream panels, are performing the same database i/o (deletes and inserts) that should be handled by the model and communicated back when done if another component cares

File: src/main/java/decodes/dbeditor/NetlistListPanel.java [hunk 1]
```diff
@@ -78,51 +54,54 @@ public class NetlistListPanel extends JPanel
...
-	/** Constructor. */
-	public NetlistListPanel()
-	{
+    /** Constructor. */
+    public NetlistListPanel()
+    {
         try
-		{
-			tableModel = new NetlistListTableModel();
-			netlistListTable = new SortingListTable(tableModel,
```
[the remark this answers is not in the document]
Comment (MikeNeilson) [reply src/main/java/decodes/dbeditor/NetlistListPanel.java:90]: Good idea.

File: src/main/java/decodes/dbeditor/presentation/PresentationGroupTableModel.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+	public void add(PresentationGroup ob)
+	{
+		pgList.add(ob);
+		fireTableDataChanged();
+	}
+
+	public void replace(PresentationGroup oldOb, PresentationGroup newOb)
+	{
+		pgList.remove(oldOb);
+		pgList.add(newOb);
```
Comment (ktarbet) [inline src/main/java/decodes/dbeditor/presentation/PresentationGroupTableModel.java:72]: should the table refresh here?

**Review by ktarbet — APPROVED** [review]

File: src/main/java/decodes/dbeditor/presentation/PresentationGroupTableModel.java (hunk 2 above)
Comment (MikeNeilson) [reply src/main/java/decodes/dbeditor/presentation/PresentationGroupTableModel.java:72]: Yep, good catch.

**Review by ktarbet — APPROVED** [review]

**Pull request merged**